readme/frontmatter/frontmatter.go

```diff
@@ -0,0 +1,77 @@
+// package frontmatter includes types and functions that provide resource plan
```

Comment:
This is the contents of the existing `frontmatter.go` copied into its own package for improved re-usability and organization.